src/BigQuery/Job.php

```diff
@@ -121,7 +127,8 @@ public function cancel(array $options = [])
     }
```

Comment:
Do you think it would be a good compromise to have calling `rows` on `QueryResults` block until the results are received?

So the options then would be:

```php
$job = $bigQuery->runQueryAsJob($query);
$results = $job->queryResults(); // let's not block here anymore...

// explicitly call waitUntilComplete
$results->waitUntilComplete(); // blocks

// or implicitly by attempting to access rows
$rows = $results->rows(); // blocks
```